### ISSUE TITLE
Built in custom problem detectors require bash

### ISSUE THREAD

**Issue by negz** [kubernetes/node-problem-detector#198]
Both built in custom problem detector scripts use `/bin/bash`, which is not installed as of version 0.5.0. This results in issues like:

```
Events:
  Type     Reason         Age                From                                                                            Message
  ----     ------         ----               ----                                                                            -------
  Warning  ConntrackFull  1s (x31 over 15m)  network-custom-plugin-monitor, tfk-negz-ing-xcb9.c.REDACTED.internal  Error in running plugin. Please check the error log
```

```
 kubectl --kubeconfig=/Users/negz/tfk-negz.kcfg -n kube-system logs node-problem-detector-vl8tw|head -n1
E0822 23:37:13.175202       1 plugin.go:115] Error in running plugin "./config/plugin/network_problem.sh": error - fork/exec ./config/plugin/network_problem.sh: no such file or directory. output - ""
```

**Comment by andyxning** [comment]
@negz Actually this has been fixed in master branch in https://github.com/kubernetes/node-problem-detector/pull/194. If you need you can build from the master branch and it should works.

**Comment by negz** [comment]
@andyxning Thanks! Just in case others find themselves here I think #194 was a typo. This bug is a duplicate of #185 and was fixed in #189.
